docs/configuration.md

```diff
@@ -1107,6 +1107,14 @@ Apart from these, the following properties are also available, and may be useful
     Compression will use <code>spark.io.compression.codec</code>.
   </td>
 </tr>
+<tr>
+  <td><code>spark.checkpoint.compress</code></td>
+  <td>false</td>
+  <td>
+    Whether to compress RDD checkpoints. Generally a good idea.
+    Compression will use <code>spark.io.compression.codec</code>
```

Comment:
Please add a `.` at the end of the sentence.

Reply:
@dongjoon-hyun added a `.` at the end of the sentence